api.ts

```diff
@@ -1972,6 +1979,14 @@ export interface SyncTableOptions<
    * sync tables that have a dynamic schema.
    */
   dynamicOptions?: DynamicOptions;
+
+  /**
+   * Used to mark a sync table as having a specific purpose
```

Comment:
We'll probably do a sweep through the tsdoc comments before we would publish them but there's always a chance we miss some of these, so I'd commit everything up to the style of our published documentation. So always complete sentences and properly punctuated.

I might do something like "Used to indicate that the entities in this table have a specific semantic meaning known to Coda, for example, that the rows being synced each represent a user."